service_mesh/v2x/upgrading-ossm-to-2-1.adoc

```diff
@@ -0,0 +1,94 @@
+[id="ossm-upgrade-to-2-1"]
+= Upgrading {ProductName} from version 2.0 to version 2.1
+include::modules/ossm-document-attributes.adoc[]
+:context: oossm-upgrading-to-2-1
+
+toc::[]
+
+Upgrading from version 2.0 to 2.1 requires manual steps that migrate your workloads and application to a new instance of {ProductName} running the new version.
+
+.Prerequisites
+
+* You must upgrade to {product-title} 4.7. before you upgrade to {ProductName} 2.0.
```

Comment:
Not a strict requirement, generally we describe prerequisites in the present tense. Should line 12 end with "2.1"? Depending on the actual meaning, here are some suggestions.
```suggestion
* You are running {product-title} 4.7.
```
```suggestion
* You upgraded to {product-title} 4.7. before upgrading to {ProductName} 2.0.
```

Reply:
Made this edit

Reply:
Dev team tweaked.